resolwe_bio/processes/samtools/samtools_bedcov.py

```diff
@@ -163,51 +133,52 @@ def run(self, inputs, outputs):
         name = f"{Path(inputs.bedfile.output.bed.path).stem}"
         coverage_report = f"{name}_coverage.txt"
```

Comment:
Maybe we should save this as a TSV file, as it is indeed tab delimited.

Reply:
I vote to retain .txt